modules/apps/transfer/ibc_module_test.go

```diff
@@ -278,31 +278,43 @@ func (suite *TransferTestSuite) TestOnTimeoutPacket() {
 	var packet channeltypes.Packet
 
 	testCases := []struct {
-		name     string
-		malleate func()
-		expError error
+		name           string
+		coinsToSendToB sdk.Coins
+		malleate       func()
+		expError       error
 	}{
 		{
 			"success",
+			sdk.NewCoins(ibctesting.TestCoin),
+			func() {},
+			nil,
+		},
+		{
+
+			"success with multiple coins",
```

Comment:
As an example:

Reply:
I am fine with both options. Still think doesn't harm to have both test cases.